edi_oca/utils.py

```diff
@@ -1,25 +1,3 @@
-# Copyright 2020 ACSONE SA
-# @author Simone Orsi <simahawk@gmail.com>
-# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+# ruff: noqa: F401
 
-import hashlib
-
-from odoo.addons.queue_job.job import identity_exact_hasher
-
-
-def normalize_string(cls, a_string, sep="_"):
-    """Normalize given string, replace dashes with given separator."""
-    return cls.env["ir.http"]._slugify(a_string).replace("-", sep)
-
-
-def get_checksum(filecontent):
-    return hashlib.md5(filecontent).hexdigest()
-
-
-def exchange_record_job_identity_exact(job_):
-    hasher = identity_exact_hasher(job_)
-    # Include files checksum
-    hasher.update(
-        str(sorted(job_.recordset.mapped("exchange_filechecksum"))).encode("utf-8")
-    )
-    return hasher.hexdigest()
+from odoo.addons.edi_core_oca.utils import get_checksum, normalize_string
```

Comment:
```suggestion
from odoo.addons.edi_core_oca.utils import get_checksum, normalize_string
try:
    from odoo.addons.edi_component_oca.utils import exchange_record_job_identity_exact
except:
    _logger.info("deprecated....")
    bla bla
```
and if we log we should log for all the imports? But in v19 we'll trash this so.... whatever